internal/provider/common.go

```diff
@@ -121,8 +121,8 @@ func ValidateImage(image string, metadata, labels map[string]string, watchByDef
 		img.Metadata = imageDefaults.Metadata
 	}
 
-	// Update metadata with metadata passed in from provider
-	img.Metadata = UpdateMap(img.Metadata, metadata)
+	// Update provider metadata with metadata from img labels
+	img.Metadata = UpdateMap(metadata, img.Metadata)
```

Comment:
@crazy-max I wasn't sure what the preferred behavior is here. I've update the branch so HEAD preserves the status quo of img labels taking priority over the provider metadata values, but it can be flipped by reverting this commit.